### PR TITLE
fix: Deduplication NPE fix

### PR DESCRIPTION
**Pull request by stian-sandvold**

### CODE REVIEW

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/deduplication/MergeObject.java [hunk 1]
```diff
@@ -43,4 +44,18 @@
     private List<String> trackedEntityAttributes;
```
Comment (enricocolasante) [inline dhis-2/dhis-api/src/main/java/org/hisp/dhis/deduplication/MergeObject.java:44]: What if we initialise the collections here?
If it works don't need of the init method

**Review by enricocolasante — APPROVED** [review]
One small comment

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/deduplication/MergeObject.java (hunk 1 above)
Comment (stian-sandvold) [reply dhis-2/dhis-api/src/main/java/org/hisp/dhis/deduplication/MergeObject.java]: Done

Comment (vilkg) [reply dhis-2/dhis-api/src/main/java/org/hisp/dhis/deduplication/MergeObject.java]: This should work, along with `@Builder.Default` on fields and `@JsonDeserialize( builder = MergeObject.MergeObjectBuilder.class)` on merge object

**Pull request merged**